modules/installation-ibm-power-vs-config-yaml.adoc

```diff
@@ -0,0 +1,354 @@
+//+
+//[IMPORTANT]
+//====
+//The use of FIPS Validated or Modules in Process cryptographic libraries is only supported on {product-title} deployments on the `x86_64` architecture.
+//====
+//<6> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
+//endif::openshift-origin[]
+
+ifdef::openshift-origin[]
+<5> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
+endif::openshift-origin[]
++
+[NOTE]
+====
+For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
+====
+endif::ibm-powervs-vpc[]
+
+ifdef::restricted[]
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+controlPlane: <2> <3>
+  hyperthreading: Enabled <4>
+  name: master
+  platform:
+  replicas: 3
+compute: <2> <3>
+- hyperthreading: Enabled <4>
+  name: worker
+  platform:
+    ibmcloud: {}
+  replicas: 3
+metadata:
+  name: example-restricted-cluster-name <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14 <5>
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16 <6>
+  networkType: OVNKubernetes <7>
+  serviceNetwork:
+  - 192.168.0.0/24
+platform:
+  powervs:
+    userid: ibm-user-id
+    powervsResourceGroup: "ibmcloud-resource-group" <8>
+    region: "powervs-region"
+    vpcRegion: "vpc-region"
+    vpcName: name-of-existing-vpc <9>
+    vpcSubnets: <10>
+       - name-of-existing-vpc-subnet
+    zone: "powervs-zone"
+    serviceInstanceID: "service-instance-id"
+publish: Internal
+credentialsMode: Manual
+pullSecret: '{"auths":{"<local_registry>": {"auth": "<credentials>","email": "you@example.com"}}}' <11>
+sshKey: ssh-ed25519 AAAA... <12>
+additionalTrustBundle: | <13>
+    -----BEGIN CERTIFICATE-----
+    <MY_TRUSTED_CA_CERT>
+    -----END CERTIFICATE-----
+imageContentSources: <14>
+- mirrors:
+  - <local_registry>/<local_repository_name>/release
+  source: quay.io/openshift-release-dev/ocp-release
+- mirrors:
+  - <local_registry>/<local_repository_name>/release
+  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+----
+<1> Required.
+<2> If you do not provide these parameters and values, the installation program provides the default value.
+<3> The `controlPlane` section is a single mapping, but the `compute` section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`, and the first line of the `controlPlane` section must not. Only one control plane pool is used.
+//<4> Enables or disables simultaneous multithreading, also known as Hyper-Threading. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
```

Comment:
@alishaIBM With this call-out commented out, it affects the numbering in the docs. 

![image](https://github.com/openshift/openshift-docs/assets/25209155/e932a032-d7c9-44de-9dfa-e628bbcd26d4)

Reply:
@mburke5678 thanks for the correction. Incorporated the changes in this PR : https://github.com/openshift/openshift-docs/pull/60086/files Thanks.